source/mobile/mobile-hpns.rst

```diff
@@ -1,44 +1,117 @@
-Hosted Push Notification Service
-================================
+Mobile Push Notifications
+=========================
 
-Securing your Mattermost Push Notification Service
---------------------------------------------------
+A push proxy is a key technology behind notification transmission that enables notifications between the server and a Mobile app. See our `Mobile Apps FAQ documentation <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-push-notifications-work>`__ to learn more about how push notifications work.
 
-When using Mattermost mobile apps in the Apple App Store and Google Play, purchase an annual subscription to `Mattermost Enterprise Edition E10 <https://mattermost.com/pricing-self-managed/>`__ or higher to receive access to our Hosted Push Notification Service (HPNS).
+Mattermost offers push proxy options for Mattermost Team Edition, Cloud, and Enterprise deployments, including:
+
+- `Test Push Notifications Service (TPNS) <#id1>`_
+- `Hosted Push Notifications Service (HPNS) <#id2>`_
+- `Mattermost Push Notification Service (MPNS) <#id3>`_
```

Comment:
This isn't an "option" afaik. MPNS is an umbrella term for the other three options (see below comment)

Reply:
Thanks! I'll rework this section.